impl/maven-impl/src/main/java/org/apache/maven/impl/resolver/scopes/Maven3ScopeManagerConfiguration.java

```diff
@@ -24,15 +24,14 @@
 import java.util.Collections;
 import java.util.stream.Collectors;
 
-import org.eclipse.aether.artifact.ArtifactProperties;
+import org.apache.maven.api.DependencyScope;
+import org.apache.maven.impl.resolver.artifact.MavenArtifactProperties;
 import org.eclipse.aether.impl.scope.BuildScopeMatrixSource;
 import org.eclipse.aether.impl.scope.BuildScopeSource;
 import org.eclipse.aether.impl.scope.CommonBuilds;
 import org.eclipse.aether.impl.scope.InternalScopeManager;
 import org.eclipse.aether.impl.scope.ScopeManagerConfiguration;
 import org.eclipse.aether.internal.impl.scope.ScopeManagerDump;
-import org.eclipse.aether.scope.DependencyScope;
-import org.eclipse.aether.scope.ResolutionScope;
```

Comment:
might redo

Reply:
and what should I do with already imported `org.apache.maven.api.DependencyScope`?